packages/example-log-extension/package.json

```diff
@@ -47,6 +47,7 @@
   "dependencies": {
     "@loopback/context": "^0.1.2",
     "@loopback/core": "^0.1.2",
+    "@loopback/openapi-v3": "^0.1.1",
```

Comment:
Can we re-export decorators like `get` and `param` from `@loopback/rest` so that LoopBack consumers don't have to upgrade when we change the underlying spec (like from Swagger to OpenAPI)?

Reply:
I am also thinking of re-exporting it or not doing that.

> LoopBack consumers don't have to upgrade when we change the underlying spec (like from Swagger to OpenAPI)

Actually I would like to encourage them to change the importing module, like how they import `@model` and `@property` from `@loopback/repository`.
And we re-exported it before to avoid breaking change, while v2-->v3 upgrade is a breaking change anyway, so if we want to decouple controller decorators and rest package, this is a proper time.

Since this could be a user experience issue, I would like to hear more opinions from the team @strongloop/loopback-next : 

Would you prefer to import controller decorators (e.g. `@get`, `@post`, `@param`, `@requestBody`) from `@loopback/rest` or `@loopback/openapi-v3`?

Reply:
I think it's better to re-export the decorators from `rest` since it simplifies the user experience by abstracting away the underlying package.

As a user trying to create REST APIs, I want to be able to use the `rest` package to do so without having to understand what OpenAPI is / what it does, etc. I just want to be able to create the api.

Reply:
+1 for what @virkt25 said 👍

Reply:
cool @virkt25 thanks for the feedback :) Added the export back.